### DISCUSSION TITLE
RFC 19 - Error Reporting

### DISCUSSION THREAD

**Discussion by jonathanyeung** [dendronhq/dendron#1121] [RFCs/Ideas]
Placeholder Discussion for feedback regarding RFC 19 - Error Reporting

**Comment by kevinslin** [comment]
> Below is an example error event that has been uploaded. Note: the local file paths in the stack trace for source maps will not appear in production; they are there in this example because I ran the test upload scenario using locally built code.

When we start working on this, we should replace this example with an actual stack trace

> The types of errors we plan on capturing data for are exceptions and some non-exception induced errors.

Can we add more details on this? eg. will caught exceptions be included? is it only uncaught exceptions? 

> some other comments

1. can we give the user an option to upload the stack trace to Dendron? by default, dendron respects your telemetry settings. on FATAL errors, can we show a dialog window with the stack trace and a button to give the user a choice if they want to upload it to Dendron?
2. can we go into detail about what workspace specific metadata will be uploaded? if any ntoe data will beuploaded? 
3. the sample error is quite large. can you link to sentry docs that explain what the various fields mean?
